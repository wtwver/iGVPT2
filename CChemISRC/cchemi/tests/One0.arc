

                     SUMMARY OF  PM3 CALCULATION, Site No: 1079

                                                       MOPAC2016 (Version: 19.338L)
                                                       Wed Mar  4 14:24:09 2020

           Empirical Formula: H2 O  =     3 atoms

 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -53.43270 KCAL/MOL =    -223.56244 KJ/MOL
          TOTAL ENERGY            =       -324.90735 EV
          ELECTRONIC ENERGY       =       -474.27017 EV
          CORE-CORE REPULSION     =        149.36282 EV
          GRADIENT NORM           =          0.65369 = 0.37741 PER ATOM
          DIPOLE                  =          1.73858 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.316247 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.058
          MOLECULAR WEIGHT        =         18.0152
          COSMO AREA              =         42.43 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.15 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53633
            O     3    H     1     0.56094
            H     1    H     2     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.020 SECONDS


          FINAL GEOMETRY OBTAINED
 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
  H     0.00000000 +1   0.77870100 +1  -0.44376600 +1
  H     0.00000000 +1  -0.75754000 +1  -0.46039600 +1
  O     0.00000000 +1   0.00472800 +1   0.10883100 +1
 
