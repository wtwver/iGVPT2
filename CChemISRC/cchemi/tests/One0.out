 *******************************************************************************
 ** Site#: 1079         For non-commercial use only    Version 19.338L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.338L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM3 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Wed Mar  4 14:24:09 2020  *
 *  PM3        - The PM3 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM = 0
 *
 *
 *
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  GRADIENTS  - ALL GRADIENTS TO BE PRINTED
 *******************************************************************************
PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          0.00000000  *   0.77870100  *  -0.44376600  *
     2       H          0.00000000  *  -0.75754000  *  -0.46039600  *
     3       O          0.00000000  *   0.00472800  *   0.10883100  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          0.0000      0.7787     -0.4438
     2         H          0.0000     -0.7575     -0.4604
     3         O          0.0000      0.0047      0.1088

  H: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).
  O: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).

           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 4


 -------------------------------------------------------------------------------
 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM3 CALCULATION
                                                       MOPAC2016 (Version: 19.338L)
                                                       Wed Mar  4 14:24:09 2020




          FINAL HEAT OF FORMATION =        -53.43270 KCAL/MOL =    -223.56244 KJ/MOL


          TOTAL ENERGY            =       -324.90735 EV
          ELECTRONIC ENERGY       =       -474.27017 EV  POINT GROUP:     C2v 
          CORE-CORE REPULSION     =        149.36282 EV
          COSMO AREA              =         42.43 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.15 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.65369 = 0.37741 PER ATOM
          IONIZATION POTENTIAL    =         12.316247 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.058
          NO. OF FILLED LEVELS    =          4
          MOLECULAR WEIGHT        =         18.0152

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53633
            O     3    H     1     0.56094
            H     1    H     2     0.00000


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.016 SECONDS

          LARGEST ATOMIC GRADIENTS

          Atom     Label                     Cartesian Gradients           Cartesian Coordinates       |Gradient|
                                           X          Y          Z        X          Y          Z

             2       H                   0.000     -0.448     -0.216    0.00000   -0.75754   -0.46040      0.497
             3       O                   0.000      0.335      0.234    0.00000    0.00473    0.10883      0.408
             1       H                   0.000      0.113     -0.018    0.00000    0.77870   -0.44377      0.115



       FINAL  POINT  AND  DERIVATIVES

   PARAMETER     ATOM    TYPE            VALUE        GRADIENT
      1          1  H    CARTESIAN X     0.000000      0.000000  KCAL/ANGSTROM
      2          1  H    CARTESIAN Y     0.778701      0.113220  KCAL/ANGSTROM
      3          1  H    CARTESIAN Z    -0.443766     -0.017897  KCAL/ANGSTROM
      4          2  H    CARTESIAN X     0.000000      0.000000  KCAL/ANGSTROM
      5          2  H    CARTESIAN Y    -0.757540     -0.447834  KCAL/ANGSTROM
      6          2  H    CARTESIAN Z    -0.460396     -0.216321  KCAL/ANGSTROM
      7          3  O    CARTESIAN X     0.000000      0.000000  KCAL/ANGSTROM
      8          3  O    CARTESIAN Y     0.004728      0.334614  KCAL/ANGSTROM
      9          3  O    CARTESIAN Z     0.108831      0.234218  KCAL/ANGSTROM




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          0.00000000  *   0.77870100  *  -0.44376600  *
     2       H          0.00000000  *  -0.75754000  *  -0.46039600  *
     3       O          0.00000000  *   0.00472800  *   0.10883100  *

                             CARTESIAN COORDINATES

   1    H        0.000000000     0.778701000    -0.443766000
   2    H        0.000000000    -0.757540000    -0.460396000
   3    O        0.000000000     0.004728000     0.108831000


           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   C2v 


                  EIGENVALUES  
 -36.82133 -17.58052 -14.52178 -12.31625   4.05767   5.33126


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          H           0.179305        0.8207     0.82069
    2          H           0.179299        0.8207     0.82070
    3          O          -0.358604        6.3586     1.76343     4.59518
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.010    -0.966     0.966
 HYBRID         0.000     0.008    -0.772     0.772
 SUM            0.000     0.019    -1.738     1.739


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  H   0.82069
    2  H   0.82070
    3  O   1.76343   2.00000   1.17964   1.41554

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.03 SECONDS

 == MOPAC DONE ==
